sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Text Analytics client library for Python
 
-Text Analytics is a cloud-based service that provides advanced natural language processing over raw text, and includes the following main features:
+The Cognitive Service for Language is a cloud-based service that provides advanced natural language processing over raw text, and includes the following main features:
```

Comment:
Not very convinced that this is true for Language service.  We could hint it and then explain that this SDK has the features XYZ, and we use the service terminology
What about something like:
```
Azure Cognitive Services Text Analytics is one of the three services that are part of the Azure Cognitive Service for Language. It includes the following main features:

* Extract Information: Use Natural Language Understanding (NLU) to extract information from unstructured text. For example:
  * Key Phrase Extraction
  * Entity Recognition (Named, Linked, and Personally Identifiable Information (PII) entities)
  * Healthcare Recognition
  * Extractive Text Summarization
  * Custom Named Entity Recognition
* Classify Text: Use Natural Language Understanding (NLU) to detect the language or classify the sentiment of text you have
  * Language Detection
  * Sentiment Analysis
  * Custom text classification
```

Reply:
I'm not comfortable grouping the features since the service itself does not do this (it's presented as a flat list). We had talked about grouping features in the past (in context of separate clients) and were advised not to. I think I will just change the intro sentence to something more general aka what is used in the service docs: "Azure Cognitive Service for Language is a cloud-based service that provides Natural Language Processing (NLP) features for _understanding and analyzing text_"

Reply:
I took it from https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/ but just intro sentence works too